susemanager/bin/mgr-setup

```diff
@@ -70,124 +57,31 @@ if [[ -n "$TZ" ]]; then
 fi
 
 TMPDIR="/var/spacewalk/tmp"
-DO_MIGRATION=0
-DO_SETUP=0
-LOGFILE="0"
-WAIT_BETWEEN_STEPS=0
-MANAGER_FORCE_INSTALL=0
 PROGRAM="/usr/lib/susemanager/bin/mgr-setup"
-NON_INTERACTIVE=0
 
-MIGRATION_ENV="/root/migration_env.sh"
 SETUP_ENV="/root/setup_env.sh"
 MANAGER_COMPLETE="/root/.MANAGER_SETUP_COMPLETE"
-MANAGER_COMPLETE_HOOK="/usr/lib/susemanager/hooks/suma_completehook.sh"
-RSYNC_LOG="/var/log/rhn/migration-rsync.log"
-
-SATELLITE_HOST=""
-SATELLITE_DOMAIN=""
-SATELLITE_DB_USER=""
-SATELLITE_DB_PASS=""
-SATELLITE_DB_SID=""
-
-SATELLITE_FQDN=""
-SATELLITE_IP=""
 
-FROMVERSION="3.2"
-KEYFILE="/root/migration-key"
-DBDUMPFILE="susemanager.dmp.gz"
-SERVER_CRT="/etc/pki/tls/certs/spacewalk.crt"
 [ -z "$SERVER_KEY" ] && { SERVER_KEY="/etc/pki/tls/private/spacewalk.key"; }
```

Comment:
I think this never worked. On line 502 of original file this was checked against `SERVER_CERT` which I believe is a typo for this CRT.

For the actual SSL setup we use `mgr-ssl-cert-setup`.